backend/remote-state/pg/backend.go

```diff
@@ -31,6 +31,13 @@ func New() backend.Backend {
 				Description: "Name of the automatically managed Postgres schema to store state",
 				Default:     "terraform_remote_state",
 			},
+
+			"skip_schema_creation": &schema.Schema{
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Description: "If set to `true`, terraform won't try to create the schema",
```

Comment:
terraform => Terraform; also, should this disambiguate and say that it won't try to create the Postgres schema? [if that is the schema it refers to]